### PR TITLE
Remove JAAS

### PR DESCRIPTION
**Pull request by efge**
https://jira.nuxeo.com/browse/NXP-27942

### CODE REVIEW

File: nuxeo-services/login/nuxeo-platform-login-digest/src/main/java/org/nuxeo/ecm/ui/web/auth/digest/DigestAuthenticator.java [hunk 1]
```diff
@@ -147,21 +148,74 @@ public void initPlugin(Map<String, String> parameters) {
...
-                for (String itemPairStr : record) {
-                    itemPairStr = StringUtils.remove(itemPairStr, QUOTE);
-                    String[] parts = itemPairStr.split(EQUAL_SEPARATOR, 2);
-                    map.put(parts[0].trim(), parts[1].trim());
-                }
-            }
-        } catch (IOException e) {
+        Map<String, String> map;
+        try {
+            map = org.apache.tomcat.util.http.parser.Authorization.parseAuthorizationDigest(new StringReader(auth));
```
Comment (kevinleturc) [inline nuxeo-services/login/nuxeo-platform-login-digest/src/main/java/org/nuxeo/ecm/ui/web/auth/digest/DigestAuthenticator.java]: Should we take care to close the `StringReader`?

Comment (efge) [reply nuxeo-services/login/nuxeo-platform-login-digest/src/main/java/org/nuxeo/ecm/ui/web/auth/digest/DigestAuthenticator.java]: It just holds a `String` so closing it doesn't bring anything.

**Comment by efge** [comment]
PR ok: https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-fguillaume/709/

**Comment by efge** [comment]
Another T&P is in progress after further cleanups in the last commit

**Comment by efge** [comment]
https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-fguillaume/710/

**Comment by efge** [comment]
T&P for last push: https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-fguillaume-3/23/

**Pull request merged**